conda/install.py

```diff
@@ -56,7 +56,7 @@ def __enter__(self):
         def __exit__(self, exc_type, exc_value, traceback):
             pass
 
-on_win = bool(sys.platform == 'win32')
+on_win = bool(sys.platform.startswith("win"))
 
```

Comment:
I don't really think this is worth arguing over.  I'm happy to revert it if need be.